pkgs/applications/science/biology/macse/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, fetchurl, jre, makeWrapper }:
+
+stdenv.mkDerivation rec {
+  pname = "macse";
+  version = "2.03";
+
+  src = fetchurl {
+    url = "https://bioweb.supagro.inra.fr/${pname}/releases/${pname}_v${version}.jar";
+    sha256 = "0jnjyz4f255glg37rawzdv4m6nfs7wfwc5dny7afvx4dz2sv4ssh";
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  dontBuild = true;
+  donUnpack = true; # not working?
+  unpackPhase = ":";
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out/share/java
+    cp -s $src $out/share/java/macse.jar
+    makeWrapper ${jre}/bin/java $out/bin/macse --add-flags "-jar $out/share/java/macse.jar"
+    runHook postInstall
+  '';
+
+  meta = with stdenv.lib; {
+    description = "Multiple Alignment of Coding SEquences";
```

Comment:
`SEquences` -> `Sequences`.

Reply:
This is actualy how it is written on their website, cause they call it MACSE and not MACS. Well I change the case of the whole sentence so it's no more disturbing ;-)

Reply:
My apologies... I figured typo but I was wrong.